dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -153,15 +158,27 @@ run_ocp() {
 }
 
 deploy_che_to_ocp() {
-    #Repull init image only if IMAGE_PULL_POLICY is set to Always
-    if [ $IMAGE_PULL_POLICY == "Always" ]; then
-        docker pull "$IMAGE_INIT"
+    #Only generate scripts and config files if CHE_GENERATE_SCRIPTS=true
+    if [ $CHE_GENERATE_SCRIPTS == true ]; then
+      echo "OCP generating temporary scripts and configuration files at ${CONFIG_DIR}/instance/config/openshift/scripts/ ."
+      #Repull init image only if IMAGE_PULL_POLICY is set to Always
+      if [ $IMAGE_PULL_POLICY == "Always" ]; then
+         docker pull "$IMAGE_INIT"
+      fi
+      #wipeout config folder
+      docker run -v "${CONFIG_DIR}":/to_remove alpine sh -c "rm -rf /to_remove/" || true
+      docker run -t --rm -v /var/run/docker.sock:/var/run/docker.sock -v "${CONFIG_DIR}":/data -e IMAGE_INIT="$IMAGE_INIT" -e CHE_MULTIUSER="$CHE_MULTIUSER" eclipse/che-cli:${CHE_IMAGE_TAG} config --skip:pull --skip:nightly
+      cd "${CONFIG_DIR}/instance/config/openshift/scripts/"
```

Comment:
`${CONFIG_DIR}/instance/config/openshift/scripts/` is used twice. Please consider moving it into a separated variable like `OPENSHIFT_SCRIPTS_FOLDER`

Reply:
Added variable OPENSHIFT_SCRIPTS_FOLDER.